source/core/CEntryPoints.cpp

```diff
@@ -501,10 +501,29 @@ VIREO_EXPORT Int32 TypeRef_Alignment(TypeRef typeRef)
     return typeRef->AQAlignment();
 }
 //------------------------------------------------------------
-VIREO_EXPORT void TypeRef_Name(TypeRef typeRef, Int32* bufferSize, char* buffer)
+VIREO_EXPORT const char* TypeRef_Name(TypeRef typeRef)
 {
+    TypeManagerScope scope(typeRef->TheTypeManager());
```

Comment:
Talked about it, but should use the same pattern as other functions, sharing the typemanager reference

Reply:
Thanks for explaining the difference :)